frontend/src/scenes/hog-functions/testing/HogFunctionTesting.tsx

```diff
@@ -316,7 +316,7 @@ function RunsFilters({ id }: { id: string }): JSX.Element {
                         formKey="configuration"
                         className="deprecated-space-y-3"
                     >
-                        <HogFunctionFilters embedded={true} />
+                        <HogFunctionFilters embedded={true} logicProps={{ id }} />
```

Comment:
This isn't the way to do this. All of these components are built to be rendered under a bound logic like the main [component does](https://github.com/PostHog/posthog/blob/fix/meikel/fix-hogfilters-component/frontend/src/scenes/hog-functions/configuration/HogFunctionConfiguration.tsx#L223).

Do this for  the HogFunctionTesting component and then it should work as expected without the prop drilling